packages/expo-firebase-analytics/android/src/main/java/expo/modules/firebase/analytics/FirebaseAnalyticsModule.java

```diff
@@ -80,13 +82,48 @@ private FirebaseAnalytics getFirebaseAnalyticsOrReject(final Promise promise) {
     return analytics;
   }
 
+  private static Bundle convertToBundle(final MapArguments mapArguments) {
```

Comment:
I wonder if this has any overlap with https://github.com/expo/expo/pull/1422 -- @wschurman

Reply:
@EvanBacon - I think you may have put the incorrect number? I can tell it's missing the last number since my recent PRs are all 1422_

Reply:
Nice find. It definitely may. Especailly if we think we're being bitten by this in other places in the client. If so it would be a good argument for calling the new library I created in that PR something other than `expo-json-utils`; maybe something like `expo-native-utils` instead.

I do see that `MapArguments` is actually our code though, so maybe the right fix would be to make this change at that layer if we think this is the general right behavior of arrays containing object. I see we put it in the `unimodules.core` though, so maybe instead that's where the new JSON stuff should live?